DI-Policies.md

```diff
@@ -0,0 +1,51 @@
+# Statement
+
+Historically, the wedding industry has been massively lacking in areas of representation and equity, especially across race, gender, and sexual orientation. The fact that interracial marriage wasn’t fully legal in the U.S. until 1967, and same sex marriage wasn’t legal until 2015, has been reflected in wedding industry practices and imagery since its beginning. And to this day, it’s painfully clear just how racist, sexist, homophobic, and generally prejudiced the industry still is.
+
+It’s up to wedding vendors and industry leaders to correct this. We have a responsibility to create spaces and content where all people feel seen, safe, and valued. 
+
+It’s important to me as a wedding business owner to not only implement policies that address these issues, but also hold myself accountable to those policies. So, I’m sharing this document publicly as one part of my own efforts to do better and be better. This not an “arrival”, but rather part of an ongoing process that will evolve and change over the lifetime of my business as I learn and iterate to create more inclusive, welcoming, and safe spaces for all folx.
+
+# Specific policies
+
+These are the specific actions I commit to taking as integral parts of how I do business moving forward. 
+
++ Listening + amplifying
+
+   + Listen to and amplify the stories and experiences of marginalized groups in the industry, and out of it.
+
+   + Speak up when witness to problematic statements, situations, and practices.
+
+   + Own my own mistakes when - not if - I make them, make reparations, and do better next time.
+
++ Regular audits of website and social media feeds
+
+   + Quarterly for website.
+
+   + Monthly for social.
+
++ Vetting subcontractors + partners
+
+   + Interview second shooters, editors, and any others I work with or hire to ensure inclusive practices and the comfort/safety of my clients.
```

Comment:
Should I list what some (or all) of the interview questions would be?

Reply:
Yes please, do list them! It's great to reassure your clients and educate people as well.

Reply:
+1!

Reply:
Will do! Again, I'll ping you back in shortly.